cores/piduino/WString.h

```diff
@@ -123,5 +123,8 @@ class StringSumHelper : public String
   StringSumHelper(double num) : String(num) {}
 };
 
+typedef char __FlashStringHelper;
+#define PROGMEM
+
```

Comment:
In the real Arduino, these are contained in "avr/pgmspace.h" ... which doesn't exist in our platform.  I had thought that we wanted to avoid "AVR" since that is a specific environment and not related to the RasPI?  What is the Pi equivalent of AVR?  ... https://en.wikipedia.org/wiki/Atmel_AVR